lib/order.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'dish'
+require 'twilio-ruby'
+
+class Order
+
+  attr_reader :order_list, :total
+
+  def initialize
+    @order_list = []
+    @total = 0
```

Comment:
I was able to remove `@total` and the attr_reader and have the tests pass - perhaps you forgot to remove the attribute reader? Why might this make the tests fail?